backend/src/axiosClient.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import { env } from './constants/env';
+
+const { authToken, uri } = env;
+
+const apiClient: AxiosInstance = axios.create({
+  baseURL: uri,
```

Comment:
URL != URI, so you should define an url. Also the requests should be in another file.